ccd-definition/CaseField/CareSupervision/manageHearingsFields.json

```diff
@@ -283,5 +301,55 @@
     "FieldType": "Label",
     "SecurityClassification": "Public",
     "Searchable": "N"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "ID": "hearingDateConfirmation",
+    "FieldType": "FixedRadioList",
+    "FieldTypeParameter": "HearingDateConfirmation",
+    "Label": "Is this correct?",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "ID": "hearingDateConfirmationLabel",
+    "Label": "The hearing date you entered is in the past",
+    "FieldType": "Label",
+    "SecurityClassification": "Public",
+    "Searchable": "N"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "ID": "hearingStartDateLabel",
+    "Label": "Start date and time",
+    "FieldType": "Text",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "ID": "hearingEndDateLabel",
+    "Label": "End date and time",
+    "FieldType": "Text",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "ID": "showStartDateLabel",
```

Comment:
Might be better named as `startDateFlag` to be consistent with other existing flags.